code/modules/jobs/job_types/other/merc_classes/enforcer.dm

```diff
@@ -0,0 +1,73 @@
+/datum/job/advclass/mercenary/enforcer
+	title = "Enforcer"
+	tutorial = "You're an exiled enforcer from an clan, robed in black, and known for wild antics and loose camaraderie, You once used your blade to shake down anyone who hasn't paid their 'protection fees', nowadays, you will fight for anyone for the right price."
+	allowed_races = list(SPEC_ID_TRITON, SPEC_ID_HUMEN)
+	outfit = /datum/outfit/mercenary/enforcer
+	category_tags = list(CTAG_MERCENARY)
+	total_positions = 5
+	cmode_music = 'sound/music/cmode/Combat_Weird.ogg'
+
+	jobstats = list(
+		STATKEY_CON = 3,
+		STATKEY_END = 1,
+		STATKEY_STR = 1,
+		STATKEY_PER = 2,
+		STATKEY_INT = -2,
+		STATKEY_SPD = -2,
+	) //7 - Statline
+
+	skills = list(
+		/datum/skill/misc/swimming = 2,
+		/datum/skill/misc/climbing = 3,
+		/datum/skill/misc/sneaking = 2,
+		/datum/skill/combat/wrestling = 3,
+		/datum/skill/combat/unarmed = 2,
+		/datum/skill/combat/swords = 4,
+		/datum/skill/combat/shields = 4,
+		/datum/skill/combat/knives = 2,
+		/datum/skill/misc/reading = 1,
+		/datum/skill/misc/athletics = 4,
+		/datum/skill/misc/medicine = 2,
+	)
+
+	traits = list(
+		TRAIT_CRITICAL_RESISTANCE,
+		TRAIT_HARDDISMEMBER,
+		TRAIT_NOPAINSTUN,
+	)
+
+/datum/job/advclass/mercenary/enforcer/after_spawn(mob/living/carbon/human/spawned, client/player_client)
+	. = ..()
+	spawned.merctype = 9
+
+/datum/outfit/mercenary/enforcer
+	name = "Gun-In"
+	var/is_leader = FALSE //does nothing except give you a cooler blade.
+
+/datum/outfit/mercenary/enforcer/pre_equip(mob/living/carbon/human/H)
+	shirt = /obj/item/clothing/shirt/undershirt/easttats
+	belt = /obj/item/storage/belt/leather
+	backr = /obj/item/storage/backpack/satchel
+	if(H.pronouns == HE_HIM || H.pronouns == THEY_THEM)
```

Comment:
clothing SHOULDN'T use pronouns. As discussed some clothes don't even have sprites for other bodytypes and snowflaking exceptions will cause confusion for new additons